website/content/v1.3/learn-more/philosophy.md

```diff
@@ -52,7 +51,7 @@ Talos has always been designed with security in mind.
 With its immutability, its minimalism, its signing, and its componenture, we are
 able to simply bypass huge classes of vulnerabilities.
 Moreover, because of the way we have designed Talos, we are able to take
-advantage of a number of additional settings, such as the recommendations of the Kernel Self Protection Project (kspp) and the complete disablement of dynamic modules.
+advantage of a number of additional settings, such as the recommendations of the Kernel Self Protection Project (kspp) and completely disabling dynamic modules.
```

Comment:
dynamic modules is not true anymore in the sense that custom modules can be loaded via machineconfig

Reply:
But they still are not dynamic, in the sense of you could run `insmod` and dynamically load a module while the system runs, yes?